publication-commons/src/test/java/no/unit/nva/publication/service/impl/TicketServiceTest.java

```diff
@@ -620,33 +621,96 @@ void shouldThrowNotFoundExceptionWhenAlienElevatedUserAttemptsToFetchPublication
     void shouldReturnEmptyTicketListWhenPublicationHasNoTickets() throws ApiGatewayException {
         var publication = persistPublication(owner, DRAFT);
         var fetchedTickets = Resource.fromPublication(publication).fetchAllTickets(resourceService)
-                                 .collect(Collectors.toList());
+            .collect(Collectors.toList());
         assertThat(fetchedTickets, is(empty()));
     }
 
     @Test
     void shouldBePossibleToCreateSeveralPublishingRequestTicketsForSinglePublication() throws ApiGatewayException {
         var publication = persistPublication(owner, DRAFT);
         assertDoesNotThrow(() -> IntStream.range(0, 2)
-                                     .boxed()
-                                     .map(ticketType -> createPersistedTicket(publication,
-                                                                              PublishingRequestCase.class))
-                                     .collect(Collectors.toList()));
+            .boxed()
+            .map(ticketType -> createPersistedTicket(publication,
+                                                     PublishingRequestCase.class))
+            .collect(Collectors.toList()));
         var ticketsFromDatabase = resourceService.fetchAllTicketsForElevatedUser(owner, publication.getIdentifier())
-                                      .collect(
-                                          Collectors.toList());
+            .collect(
+                Collectors.toList());
         assertThat(ticketsFromDatabase, allOf(hasSize(2), everyItem(instanceOf(PublishingRequestCase.class))));
     }
 
+    @ParameterizedTest(name = "ticket type:{0}")
+    @DisplayName("should persist updated ticket assignee when ticket assignee is updated")
+    @MethodSource("ticketTypeProvider")
+    void shouldPersistUpdatedAssigneeWhenTicketAssigneeIsUpdated(Class<? extends TicketEntry> ticketType)
+        throws ApiGatewayException {
+        var publicationStatus = validPublicationStatusForTicketApproval(ticketType);
+        var publication = persistPublication(owner, publicationStatus);
+        var persistedTicket = createUnpersistedTicket(publication, ticketType).persistNewTicket(ticketService);
+        var user = UserInstance.fromTicket(persistedTicket);
+        ticketService.updateTicketAssignee(persistedTicket, user.getUser());
+        var updatedTicket = ticketService.fetchTicket(persistedTicket);
+
+        var expectedTicket = persistedTicket.copy();
+        expectedTicket.setAssignee(UserInstance.fromPublication(publication).getUser());
+        expectedTicket.setModifiedDate(updatedTicket.getModifiedDate());
+
+        assertThat(updatedTicket, is(equalTo(expectedTicket)));
```

Comment:
Kanskje benytte assertThat(updatedTicket.getAssignee(), is(equalTo(expectedAssignee))?

Nå settes veldig mange attributter til expectedTicket til actualTicket?